raw-span-constants/src/main/proto/org/hypertrace/core/span/constants/v1/span_attribute.proto

```diff
@@ -92,6 +92,23 @@ enum Grpc {
     GRPC_METHOD = 10 [(string_value) = "grpc.method"];
 }
 
+enum Rpc {
```

Comment:
Same here. Every constant here should be local to this service and all these should be first class fields on the span right?

Reply:
All of these are named fields in Grpc event.